docs/adguard-for-android/solving-problems/outbound-proxy.md

```diff
@@ -121,6 +121,21 @@ You should remove Shadowsocks application from filtering before setting up the p
 
 4. Enable the main Proxy switch and AdGuard protection to route your device's traffic through the proxy.
 
+### How to use AdGuard with Clash
+
+*Here we presume that you are already a Clash client and have it installed on your device.*
+
+1. Open Clash and go to **Settings** → **Network** → **Route System Traffic** and disable it. This will put Clash into proxy mode.
+2. Open Adguard and go to **Settings** → **Filtering** → **Apps** → **App management**. Choose **Clash For Android** and disable **Route traffic through Adguard**. In this way you will eliminate traffic looping.
```

Comment:
Adguard!

Reply:
In this way you will eliminate traffic looping.
—>
This will eliminate traffic looping.